### PR TITLE
Instructions page

### PR DESCRIPTION
**Pull request by sooble13**
This pr closes https://github.com/UNLV-CS472-672/2024-S-GROUP1-Roadwatch/issues/8.

### **Motivation and Context**
This pull request introduces the Instructions page to our Roadwatch application. It features the new SVG logo designed by Juan, adopts Dafne's mobile application styling, utilizes Aligary's custom button component, and integrates MUI's carousel component. These enhancements ensure the clear presentation of instructions and guidelines for using Roadwatch, providing users with a comprehensive understanding of our application's functionalities and features.

It's important to note that this isn't the finished webpage since there are no formal instruction images as of yet. Instead, there are placeholder images that will serve in its place. In addition, the custom button points to the login page for now since at the time there was no home page available when I made this.

### **Types of changes**
- [x]  Roadwatch svg logo
- [x] Aligary's custom button component
- [x] MUI's carousel component

### **Things to update**
- [ ] Sizing and styling if it's an issue.
- [ ] Replace images with proper instruction images.
- [ ] Appropriate routing of the custom button. 

### **iPhone SE**
iphonese

### **iPhone 14 Pro Max**
iphone14promax

### CODE REVIEW

**Review by dafneMena — APPROVED** [review]
Hi so I think the format looks pretty good! I noticed you talked about adding proper descriptions, images, and perhaps fixing the sizing of stuff. 

Just adding one more thing to the updates: 
Test 'Let's Go' button and make sure it points to the correct page. I'm assuming it will eventually lead to the Home page.

**Comment by valdea13** [comment]
> Hi so I think the format looks pretty good! I noticed you talked about adding proper descriptions, images, and perhaps fixing the sizing of stuff.
> 
> Just adding one more thing to the updates: Test 'Let's Go' button and make sure it points to the correct page. I'm assuming it will eventually lead to the Home page.

That's a good point! I forgot to mention that the button currently points to your login page. It was supposed to be the home page, but at the time of making this, the home page wasn't available yet. Thanks for the input!

File: client/src/pages/Instructions/Instructions.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import Styles from './Instructions.module.scss';
+import Carousel from 'react-material-ui-carousel';
+import { CustomButton } from '@/components';
+import { useNavigate } from 'react-router-dom';
+import logo from '../../assets/Updated_RoadWatch_Logo.svg';
+//import image1 from './images/image1.svg';
+//import image2 from './images/image2.svg';
+//import image3 from './images/image3.svg';
```
Comment (Linkaloo) [inline client/src/pages/Instructions/Instructions.tsx:8]: can you remove all the unused images

**Review by Linkaloo — APPROVED** [review]
Looks good, just remove unused code comments, we can add stuff if needed later

File: client/src/pages/Instructions/Instructions.tsx (hunk 1 above)
Comment (valdea13) [reply client/src/pages/Instructions/Instructions.tsx]: They are now removed.

**Review by Aligary — APPROVED** [review]
Thanks for adding this in. Now our users will have an idea on how to use our app! (once we add the actual instructions)

**Pull request merged**